packages/cozy-client/src/CozyClient.js

```diff
@@ -369,16 +369,19 @@ class CozyClient {
    */
   static fromEnv(envArg, options = {}) {
     const env = envArg || (typeof process !== 'undefined' ? process.env : {})
-    const { COZY_URL, COZY_CREDENTIALS, NODE_ENV } = env
+    const { COZY_URL, COZY_CREDENTIALS } = env
     if (!COZY_URL || !COZY_CREDENTIALS) {
       throw new Error(
         'Env used to instantiate CozyClient must have COZY_URL and COZY_CREDENTIALS'
       )
     }
-    if (NODE_ENV === 'development') {
-      options.oauth = JSON.parse(COZY_CREDENTIALS)
-    } else {
-      options.token = COZY_CREDENTIALS.trim()
+
+    if (COZY_CREDENTIALS) {
```

Comment:
We can remove this check since we already do it in https://github.com/cozy/cozy-client/pull/1245/files#diff-9d26647e8d41ba1414842f5d3dca72250075f14efed52997e19584197a66e2a6R373